Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/debuglog/DebugLogger.kt

```diff
@@ -165,26 +165,37 @@ class DebugLogger(
                     delay(1000)
 
                     val formattedMessage = rawLine.format()
-                    val censored: Collection<BugCensor.CensoredString> = bugCensors.get()
+                    val censored: Collection<BugCensor.CensorContainer> = bugCensors.get()
                         .map {
                             async {
-                                it.checkLog(formattedMessage)
+                                try {
+                                    it.checkLog(formattedMessage)
+                                } catch (e: Exception) {
+                                    Log.e(TAG, "Error in censor module $it", e)
+                                    null
```

Comment:
Wouldn't it be safer to return some String like `<censoring-error>` here? Returning `null` here means that no censoring happened and private data could end up in the log?

Reply:
I agree, will change.